src/js/handlers/flow-handler.js

```diff
@@ -45,14 +53,16 @@ export function flowEvents(){
 				appContainer.appendChild( playersScreen() );
 				appContainer.removeChild( currentScreen );				
 
-				// Used setTimeOut() here because otherwise the event below would trigger before players-screen was loaded
+				// Used setTimeOut() here because otherwise the event below would trigger before settings-screen was loaded
```

Comment:
Take a look at this https://developer.mozilla.org/en-US/docs/Web/API/Window/load_event 

See if you can use an event listener on this event (or a similar one) instead of a setTimeout